src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -4618,6 +4611,33 @@
 		<call-macro name="dctSubject" field="937"/>
 		<call-macro name="dctSubject" field="942"/>
 		<call-macro name="dctSubject" field="947"/>
+
+		<combine name="http://purl.org/dc/terms/subject" value="_:710${a}" flushWith="710[-abcdfz][123].a">
+			<data source="710[-abcdfz][123].a" name="a">
+				<regexp match="^(?!http).*$"/>
+				<replace pattern="&gt;&gt;" with=""/>
+				<replace pattern="&lt;&lt;" with=""/>
```

Comment:
the 'replace pattern="$pattern...' does in fact replaces _all_ $pattern. As we don't want to have any '<' or '>' (since this would destroy triple URIs) it's better - and simpler - to do this:
\<replace pattern="&gt;" with=""/\>